modelling/src/main/java/org/axonframework/modelling/command/inspection/AbstractChildEntityDefinition.java

```diff
@@ -95,9 +95,9 @@ private ForwardingMode instantiateForwardingMode(Field field,
                                                      Class<? extends ForwardingMode> forwardingModeClass) {
         ForwardingMode forwardingMode;
         try {
-            forwardingMode = forwardingModeClass.newInstance();
+            forwardingMode = forwardingModeClass.getDeclaredConstructor().newInstance();
```

Comment:
I believe the `Class#newInstance` call has been deprecated in Java 9 (as can be found [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Class.html#newInstance())).
The framework is however based on Java 8 at the moment, so I do not feel this change should be made at this point.

Additionally, if I perform this change on an Axon project using Java 8, this line breaks as the `getDeclaredConstructor()` expects parameter types to be inserted.

Reply:
Disregard this, I am having a bad case of the Mondays (aka, unable to correctly read exceptions from my IDE).